hivemind/p2p/p2p_daemon.py

```diff
@@ -134,11 +128,11 @@ async def create(cls, *args, quic: bool = True, tls: bool = True, conn_manager:
         return self
 
     @classmethod
-    async def replicate(cls, daemon_listen_port: int, host_port: int):
+    async def replicate(cls, daemon_listen_port: int, p2p_port: int) -> 'P2P':
         """
         Connect to existing p2p daemon
         :param daemon_listen_port: port for connection daemon and client binding
-        :param host_port: port for p2p network
+        :param p2p_port: port for p2p network
```

Comment:
[nit] see reasoning above
```suggestion
        :param p2p_port: port for external connections from other p2p instances
```